scripts/info.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e
+
+version='0.0.1'
+cb='```'
+
+if [ ! -x "$(command -v cmake)" ]; then
+  echo "Please install cmake to use this script."
+  exit 1
+fi
+
+info=$(cmake --system-information)
+
+ext() {
+  grep -oP '\".+\"$' $1 | tr -d '\"' 
+}
+
+print_info() {
+echo "$info" | grep -P "$1" | ext 
+}
+
+result="
+Velox System Info v${version}
+Commit: $(git rev-parse HEAD 2> /dev/null || echo "Not in a git repo.")
+CMake Version: $(cmake --version | grep -oP '\d+\.\d+\.\d+')
+System: $(print_info 'CMAKE_SYSTEM \"')
+Arch: $(print_info 'CMAKE_SYSTEM_PROCESSOR')
+C++ Compiler: $(print_info 'CMAKE_CXX_COMPILER ==')
+C++ Compiler Version: $(print_info 'CMAKE_CXX_COMPILER_VERSION')
+C Compiler: $(print_info 'CMAKE_C_COMPILER ==')
+C Compiler Version: $(print_info 'CMAKE_C_COMPILER_VERSION')
+CMake Prefix Path: $(print_info '_PREFIX_PATH ')
```

Comment:
Details in what way? Just `ls /usr/local/lib`? And should we use an absolute path or just the first prefix path? Or all of the prefix  paths?

Reply:
Just ls I suppose for all prefix paths. Lets do one thing, lets gate that via some flag (say additional-details), which we can request if the initial paste doesnt help.

Reply:
I have tested this a bit and don't think that this is something we should do automated. It either gives too little information (`ls` without recursing into dirs) or **way too much** information as it potentially lists the complete contents of the users fs because `/` is part of `CMAKE_PREFIX_PATHS`.

I think it would be better to just ask the user in those cases to list this manually.